configs/cqrs_config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Shoudn't we put this under the dev folder?

Reply:
What do you think of using `testdata` naming, because if I got it well it should be ignored by the _Go_ tooling